tests/markets/seer/test_price_manager.py

```
@@ -0,0 +1,159 @@
+                Web3.to_checksum_address(token) for token in mock_market.wrapped_tokens
+            ],
+        )
+
+    # Verify all keys in outcome token pool are stripped and lowercase
+    assert all(key == key.strip().lower() for key in outcome_token_pool.keys())
+    assert set(outcome_token_pool.keys()) == {"yes", "no", "invalid"}
+    assert all(isinstance(key, str) for key in outcome_token_pool.keys())
+    assert all(isinstance(val, OutcomeToken) for val in outcome_token_pool.values())
+
+    # Verify all keys in probability map are stripped and lowercase
+    assert all(key == key.strip().lower() for key in probability_map.keys())
```

Ruff (0.12.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)